Yoast/wordpress-seo
# Remove taxonomy-meta css because is isn't used

## Summary

This PR can be summarized in the following changelog entry: n/a

## Relevant technical choices:

* The risk is low because there is only one line of CSS in the file.

## Test instructions

This PR can be tested by following these steps:

* Make sure no view is horrible broken by this CSS change.

It looks like it was used before, but currently there is no enqueue in
the code that actually includes this CSS file.
